crates/types/src/config/worker.rs

```diff
@@ -68,6 +68,8 @@ pub struct WorkerOptions {
     ///
     /// Snapshots provide a mechanism for safely trimming the log and efficient bootstrapping of new
     /// worker nodes.
+    #[serde(skip_serializing_if = "is_default_snapshots_options")]
```

Comment:
Thanks so much for hiding these! This has been bothering me but I didn't know how to fix it 🙏